docs/src/appendix/sncast/account/import.md

```diff
@@ -17,7 +17,7 @@ Address of the account.
 ## `--type, -t <ACCOUNT_TYPE>`
 Required.
 
-Type of the account. Possible values: oz, argent, braavos.
+Type of the account. Possible values: `oz`, `ready`, `braavos`.
```

Comment:
Maybe it'd make sense to say `formerly bravos` here as well? And in the code `--help`.

Reply:
`formerly argent`, then yes 😆